test/Npgsql.Tests/SystemTransactionTests.cs

```diff
@@ -269,7 +417,7 @@ public void ReuseConnection()
         [Test]
         public void ReuseConnectionRollback()
```

Comment:
This test was looking like a copy paste of the previous one, renamed but not changed. Changes done now.